pkgs/by-name/op/opentofu/package.nix

```diff
@@ -53,8 +58,17 @@ let
       installShellCompletion --bash --name tofu <(echo complete -C tofu tofu)
     '';
 
+    __darwinAllowLocalNetworking = true;
+
+    nativeCheckInputs = [
+      writableTmpDirAsHomeHook
+      versionCheckHook
+    ];
+
+    doInstallCheck = true;
+    versionCheckProgramArg = "version";
```

Comment:
Can explain this in short? Why is __darwinAllowLocalNetworking needed and the rest.